Start-C4bVerification.ps1

```diff
@@ -0,0 +1,36 @@
+[CmdletBinding()]
+Param(
+    [Parameter(Mandatory)]
+    [String]
+    $Fqdn
+)
+
+process {
+    Write-Host "Installing Pester 5 to run validation tests"
+    $chocoArgs = @('install', 'pester', '-y', '--source="https://community.chocolatey.org/api/v2/"')
```

Comment:
Should we specify version 5.3.1 of the pester package in this array? Mainly ask as i know Pester 6 was just announced recently.

Reply:
I'm gonna act like i didn't just ask this.

Reply:
Yes, but not because Pester 6 was announced 🤪  Ideally we'd pin to 5.99.99 or something like that, but I think it's reasonable to go with 5.3.1 since it's known to work, and we can increment it as new versions are available.